frontend/src/app/shared/components/editor/components/ckeditor/ckeditor-setup.service.ts

```diff
@@ -60,7 +60,7 @@ export class CKEditorSetupService {
     return this
       .createWatchdog(editorClass, contentWrapper, config)
       .then((watchdog:ICKEditorWatchdog) => {
-        const editor = watchdog.editor;
```

Comment:
I have no possibility to test this right now. @oliverguenther does the watchdog feature still work with this changes?